raaghu-elements/src/rds-card/rds-card.stories.tsx

```
@@ -723,11 +783,20 @@ export const CardWithLineChart: Story = {
+    state: "Default",
+    style: "Default",
+    showIndicator: false,
+    colorVariant: "primary",
+    borderColor: "",
+    isDisabled: false,
+};
 
 export const CardWithDataTable: Story = {
   args: {
-    ...commonArgs,
+    ...cardWithDataTableArgs,
```

ESLint

[error] Expected indentation of 8 spaces but found 4.

(indent)